lrpar/tests/build.rs

```diff
@@ -0,0 +1,57 @@
+use cfgrammar::yacc::{YaccKind, YaccOriginalActionKind};
+use glob::glob;
+use lrlex::LexerBuilder;
+use lrpar::CTParserBuilder;
+use std::{env, fs, io::Read, path::PathBuf};
+use yaml_rust::YamlLoader;
+
+fn main() -> Result<(), Box<std::error::Error>> {
+    let out_dir = env::var("OUT_DIR").unwrap();
+    for entry in glob("src/*.test")? {
+        match entry {
```

Comment:
Since the `Err` path is `panic`, I think you can remove the `match` and just say `let path = entry.unwrap()`?